kubernetes/ccloudexporter.svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ccloudexporter
+    ccloudexporter: ccloudexporter
+  name: ccloudexporter
+  #annotations:
```

Comment:
Shouldn't those annotations be un-commented?

Reply:
It can be deleted. Annotations are not accepted by the Prometheus Operator, only ServiceMonitor for servide discovery.